src/main/kotlin/org/aarbizu/baseballDatabankFrontend/db/QueryEngine.kt

```diff
@@ -43,12 +39,11 @@ class QueryEngine(private val dbProvider: DBProvider) {
         ): List<TableRecord> {
             val timer = Stopwatch.createStarted()
             val rs = statement(dbProvider, queryTemplate, binds).executeQuery()
-            timer.reset()
             logger.info("exec query: $timer")
```

Comment:
I think the current version will have the first `logger.info` always print a zeroed (reset) timer...